### PR TITLE
Magento circle ci

### PR DESCRIPTION
**Pull request by ethanwayda22**
# Description
Add docker integration test for future commits. Each time a pr is made we can be sure it does not break the flow.

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [x] Successfully tested locally (or docker image)
- [x] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] New and existing unit tests pass locally with my changes
- [ ] I have created or modified unit tests to sufficiently cover my changes.

### CODE REVIEW

**Comment by ethanwayda22** [comment]
previous comments can be found here, needed to incorporate new docker changes into this branch
https://github.com/BoltApp/bolt-magento2/pull/342

**Comment by ethanwayda22** [comment]
Built worked just canceled for ssh testing https://circleci.com/workflow-run/48f5d8b7-2f53-44e2-b330-493a1664c23c

File: Test/scripts/ci-integration.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+php bin/magento module:enable Bolt_Boltpay
+
+# # set config
+php bin/magento config:set payment/boltpay/active 1
+php bin/magento config:set payment/boltpay/api_key $BOLT_STAGING_MERCHANT_API_KEY
+php bin/magento config:set payment/boltpay/signing_secret $BOLT_STAGING_MERCHANT_SIGNING_SECRET
+php bin/magento config:set payment/boltpay/publishable_key_checkout $BOLT_STAGING_MERCHANT_PUBLISHABLE_KEY
+
+# install and run ngrok
+wget -O ngrok.zip https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-amd64.zip
```
Comment (roopakv) [inline Test/scripts/ci-integration.sh]: lets not download a random binary and instead save this in our bucket somewhere.

File: Test/scripts/install_magento_integrations.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+#*/
+
+set -e
+set -u
+set -x
+
+trap '>&2 echo Error: Command \`$BASH_COMMAND\` on line $LINENO failed with exit code $?' ERR
+
+composer show -i
+# echo "{\"http-basic\":{\"repo.magento.com\":{\"username\":\"${MAGENTO_PUBLIC_KEY}\",\"password\":\"${MAGENTO_PRIVATE_KEY}\"}}}" > $HOME/.composer/auth.json
```
Comment (roopakv) [inline Test/scripts/install_magento_integrations.sh]: nuke these comments?

File: operations/docker/php72-integrations/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Dockerfile
```
Comment (roopakv) [inline operations/docker/php72-integrations/Dockerfile]: auth sample

**Pull request merged**